src/logmessage.cpp

```diff
@@ -122,7 +122,11 @@ namespace g3 {
 
 
    std::string LogMessage::timestamp(const std::string& time_look) const {
-      return  localtime_formatted(_timestamp, time_look);
+      std::ostringstream stringStream;
+
+      stringStream << _timestamp.tv_nsec;
+
+      return g3::localtime_formatted(_timestamp.tv_sec, (time_look.empty() ? g3::internal::date_formatted + " " + g3::internal::time_formatted : time_look)) + "|" + stringStream.str() + "|";
```

Comment:
I can see that travis CI is failing to recognize TIME_UTC macro, so I reverted back to clock_gettime